### ISSUE TITLE
SSL error: unexpected eof while reading (in ) on browser refresh of TLS crossbar connection

### ISSUE THREAD

**Issue by antonymott** [crossbario/crossbar#2047]
Browser (autobahn.js client) securely connects and generally functions OK over TLS connection, except on each browser refresh (chrome on debian 10):

SSL error: unexpected eof while reading (in )

Perhaps it's related to another SSL error I was able to reproduce over here if I try to add a container component to the config: https://github.com/pyca/cryptography/issues/7488

**Comment by antonymott** [comment]
I did some tests to see if I can help narrow down the problem. I tried to securely connect a python client, same error. If I had to guess, although the problem appears as a crossbar router problem, it's probably in the Twisted parts of Crossbar, in the way in which openssl is used by Twisted.

The TLS for web works fine, in contrast to wss, and https://<domain> passed all external TLS tests. So the problem seems narrowly wss. 

I can't spend more time on it at the moment. My workaround is to proxy a non-TLS crossbar instance behind NGINX documented in many places, like [](https://stackoverflow.com/questions/12102110/nginx-to-reverse-proxy-websockets-and-enable-ssl-wss).
